### PR TITLE
Tickets/dm 14365

### PR DESCRIPTION
**Pull request by kennywlo**

### CODE REVIEW

File: bin/server.py [hunk 1]
```diff
@@ -32,13 +32,13 @@
 import json
 import logging as log
 import sys
-from lsst.dax.metaserv import api_v0, api_v1
+from lsst.dax.metaserv import metaREST_v1
```
Comment (brianv0) [inline bin/server.py:35]: Will you keep the original name?

File: python/lsst/dax/metaserv/metaREST_v1.py [hunk 2]
```diff
@@ -40,10 +41,11 @@
...
 SAFE_SCHEMA_PATTERN = re.compile(SAFE_NAME_REGEX)
 SAFE_TABLE_PATTERN = re.compile(SAFE_NAME_REGEX)
-ACCEPT_TYPES = ['application/json', 'text/html']
 
-metaserv_api_v1 = Blueprint('metaserv_v1', __name__,
-                            template_folder="templates")
+ACCEPT_TYPES = ["application/json", "text/html"]
+
+metaRESTv1 = Blueprint("metaRESTv1", __name__, static_folder="static",
+        template_folder="templates")
```
Comment (brianv0) [inline python/lsst/dax/metaserv/metaREST_v1.py]: Please keep the original name. Also, the aligning of the second line isn't pep8

**Review by brianv0 — CHANGES_REQUESTED** [review]

File: bin/server.py (hunk 1 above)
Comment (kennywlo) [reply bin/server.py:35]: Will revert.

File: python/lsst/dax/metaserv/metaREST_v1.py (hunk 2 above)
Comment (kennywlo) [reply python/lsst/dax/metaserv/metaREST_v1.py]: will revert.

**Pull request merged**